src/app.tsx

```diff
@@ -13,18 +13,40 @@ html, body {
 }
 `
 
-export default class App extends React.Component {
-  constructor(props) {
+export type DisplayRepo = {
+  description: string,
+  language?: string,
+  name: string,
+  stars: number,
+  url: string,
+}
+
+type ResponseRepo = {
+  archived: boolean,
+  description: string,
+  fork: boolean,
+  html_url: string,
+  language?: string,
+  name: string,
+  stargazers_count: number,
+}
+
+type State = {
+  repos: Array<DisplayRepo>
+}
+
+export default class App extends React.Component<{}, State> {
+  constructor(props: any) {
```

Comment:
is this not a lint error?

Reply:
This isn't using our lint config, it's using the built in CRA one I guess